bundles/org.openhab.automation.jsscripting/pom.xml

```diff
@@ -142,33 +137,8 @@
       <artifactId>js</artifactId>
       <version>${graal.version}</version>
     </dependency>
-    <!-- com.ibm.icu.icu4j/69.1 is not required on GraalJS >= 22.0.0 as it moved to org.graalvm.truffle -->
-
-    <!-- include as version required is older than OH provides -->
-    <dependency>
-      <groupId>org.ow2.asm</groupId>
-      <artifactId>asm</artifactId>
-      <version>${asm.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.ow2.asm</groupId>
-      <artifactId>asm-commons</artifactId>
-      <version>${asm.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.ow2.asm</groupId>
-      <artifactId>asm-tree</artifactId>
-      <version>${asm.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.ow2.asm</groupId>
-      <artifactId>asm-util</artifactId>
-      <version>${asm.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.ow2.asm</groupId>
-      <artifactId>asm-analysis</artifactId>
-      <version>${asm.version}</version>
-    </dependency>
+    <!-- GraalJS changelog says that com.ibm.icu/icu4j is not required for GraalJS >= 22.0.0 as it moved to org.graalvm.truffle;
+      but GraalJS >= 22.2.0 requires it -->
+    <!-- org.ow2.asm is not required as compile dependency for GraalJS >= 20.3.0 -->
```

Comment:
Do we need these comments?  Its seems out of place since the config it references is not longer there.

Reply:
We can remove the last one, but we should keep the first one as reminder when we (hopefully) upgrade to 22.3.0 one day.